specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-01-01/apimapis.json

```diff
@@ -124,7 +124,7 @@
             "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
           },
           {
-            "$ref": "#/parameters/ApiIdParameter"
+            "$ref": "#/parameters/ApiIdRevParameter"
```

Comment:
>ApiIdRevParameter [](start = 34, length = 17)

this parameter matches the `GET` parameter for the same path. It was an issue in the old spec.